docs/source/examples/Layout and Styling.ipynb

```diff
@@ -0,0 +1,1892 @@
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget": "2fc350f07933464c86a1738c31a7237f"
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "items_layout = Layout(flex='1 1 auto', width='auto')     # override the default width of the button to 'auto' to let the button grow\n",
+    "\n",
+    "items = [\n",
+    "    Button(description='weight=1'),\n",
+    "    Button(description='weight=2', layout=Layout(flex='2 1 auto', width='auto')),\n",
+    "    Button(description='weight=1'),\n",
+    " ]\n",
+    "\n",
+    "box_layout = Layout(display='flex',\n",
+    "                    flex_flow='row', \n",
+    "                    align_items='stretch', \n",
+    "                    border='solid',\n",
+    "                    width='50%')\n",
+    "box = Box(children=items, layout=box_layout)\n",
+    "box"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**A more advanced example: a reactive form.**\n",
+    "\n",
+    "The form is a `VBox` of width '50%'. Each row in the VBox is an HBox, that justifies the content with space between.."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget": "9d55c6ae2ffd45bc9b11f08cb1119e2b"
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "label_layout = Layout()\n",
+    "\n",
+    "form_item_layout = Layout(\n",
+    "    display='flex',\n",
+    "    flex_flow='row',\n",
+    "    justify_content='space-between'\n",
+    ")\n",
+    "\n",
+    "form_items = [\n",
+    "    Box([Label(value='Age of the captain'), IntSlider(min=40, max=60)], layout=form_item_layout),\n",
+    "    Box([Label(value='Egg style'), \n",
+    "         Dropdown(options=['Scrambled', 'Sunny side up', 'Over easy'])], layout=form_item_layout),\n",
+    "    Box([Label(value='Ship size'), \n",
+    "         FloatText()], layout=form_item_layout),\n",
+    "    Box([Label(value='Information'), \n",
+    "         Textarea()], layout=form_item_layout)\n",
+    "]\n",
+    "\n",
+    "form = Box(form_items, layout=Layout(\n",
+    "    display='flex',\n",
+    "    flex_flow='column',\n",
+    "    border='solid 2px',\n",
+    "    align_items='stretch',\n",
+    "    width='50%'\n",
+    "))\n",
+    "form"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": []
+  }
```

Comment:
Are the three empty code cells at the end of the notebook (above) here on accident?